draft-ietf-quic-qpack.md

```diff
@@ -229,8 +229,9 @@ table that it will emit a reference for.  As new entries are inserted, the
 encoder increases the draining index to maintain the section of the table that
 it will not reference.  If the encoder does not create new references to entries
 with an absolute index lower than the draining index, the number of
-unacknowledged references to those entries will eventually become zero, allowing
-them to be evicted.
+unacknowledged references to every such entry will eventually become zero,
+allowing it to be evicted provided that its insertion has been acknowledged by
+the decoder.
```

Comment:
This is now a pretty long sentence.  Maybe end it after 'become zero', and start a new sentence talking about eviction?

Reply:
Ah, let me just revert the change to this paragraph.  No matter how I try to reword it, mentioning acknowledgement of the insertion always leaves it more confusing that the original text.  After all, if the number of unacknowledged references _becomes_ zero, that means it was not zero beforehand, therefore the encoder knows for sure that the insertion has been processed by the decoder.